pkgs/tools/package-management/packagekit/default.nix

```diff
@@ -1,57 +1,53 @@
-{ stdenv, fetchurl, intltool, glib, pkgconfig, polkit, python, sqlite }:
+{ stdenv, fetchgit, intltool, glib, pkgconfig, polkit, python, sqlite, systemd
+, gobjectIntrospection, vala, gtk_doc, autoreconfHook, autoconf-archive
+, nixUnstable, boost
```

Comment:
I think it's common practice to import nix and override from all-packages.nix: `callPackage ... { nix = nixUnstable; };`